qiskit/circuit/library/__init__.py

```diff
@@ -170,27 +289,18 @@
     diagonal = DiagonalGate([1, 1, 1, -1])
     print(diagonal.num_qubits)
 
+which prints:
+
 .. code-block:: text
 
     1
     2
 
-
 .. autosummary::
    :toctree: ../stubs/
-   :template: autosummary/class_no_inherited_members.rst
 
-   Diagonal
    DiagonalGate
-   MCMT
-   MCMTVChain
-   Permutation
    PermutationGate
-   GMS
-   GR
-   GRX
-   GRY
-   GRZ
    MCMTGate
    MCPhaseGate
    MCXGate
```

Comment:
should MCXGrayCode, MCXRecursive and MCXVChain be removed from here as well? and moved to the table below ?

Reply:
whoops I missed these -- thanks!